tests/test_data_streams.py

```diff
@@ -1,4 +1,5 @@
 import pickle
```

Comment:
do we still need pickle?

Reply:
Replaced with `cbor2.load`, removed `import pickle`